reactive/kubernetes_master_worker_base.py

```diff
@@ -102,17 +101,19 @@ def manage_docker_opts(opts, remove=False):
     :param: dict opts: option keys/values; use None value if the key is a flag
     :param: bool remove: True to remove the options; False to add them
     '''
-    docker_opts = DockerOpts()
-    for k, v in opts.items():
-        # Always remove existing option
-        if docker_opts.exists(k):
-            docker_opts.pop(k)
-        if not remove:
-            docker_opts.add(k, v)
-    hookenv.log('DockerOpts daemon options changed. Requesting a restart.')
-    # State will be removed by layer-docker after restart
-    set_state('docker.restart')
-
+    docker = endpoint_from_flag('endpoint.docker.ready')
+    docker_opts = docker.get('options')
```

Comment:
Where does `ep.docker.ready` come from?  It's not clear to me that this is going to get the same host options that we'd get with `charms.docker.DockerOpts`.